lib/internal/bootstrap_node.js

```diff
@@ -108,7 +108,8 @@
         'DeprecationWarning', 'DEP0062', startup, true);
     }
 
-    if (process.binding('config').experimentalModules) {
+    if (process.binding('config').experimentalModules ||
+        process.binding('config').experimentalVMModules) {
       process.emitWarning(
```

Comment:
this warning should still only show up for `--experimental-modules`

Reply:
This means that the original way the warning was written is ok?

Reply:
@punteek, I think @devsnek meant

```js
if (experimentalModules || experimentalVMModules) {
  if (experimentalModules)
    process.emitWarning(...);
  NativeModule.require(...);
}
```